nilearn/decomposition/__init__.py

```diff
@@ -3,5 +3,6 @@
 variant of the ICA called Canonical ICA.
 """
 from .canica import CanICA
+from .multi_pca import session_pca
 
-__all__ = ['CanICA']
+__all__ = ['CanICA', 'session_pca']
```

Comment:
As I feel that this is internal, I'd rather have it not imported in the __init__.